### DISCUSSION TITLE
[Idea] Add mobile options for image slides in ps_imageslider

### DISCUSSION THREAD

**Discussion by mparvazi** [PrestaShop/PrestaShop#32270] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

The ps_imageslider uses only an image for all media screens (desktop and mobile).

### Describe the solution you'd like

We can add some options for mobile screen:

**1. Mobile options is OFF (so we use desktop options in mobile screen)**

![Screenshot 2022-02-04 at 13-09-44 Module Manager • Wishlist](https://user-images.githubusercontent.com/85633460/152508677-79a1d9ce-d4c0-4461-a5a0-266de0a44e2b.png)

**2. Mobile options is ON**

![Screenshot 2022-02-04 at 13-17-51 Module Manager • Wishlist](https://user-images.githubusercontent.com/85633460/152508873-13a41fd2-b5e2-4b68-9936-5331a282e05a.png)

- User can upload an image for mobile screen
- User can control the captions placement on slider (Overlay, Top, Bottom)
- User can disable captions for mobile screens

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by HanaRebaiQA** [comment]
Hello @mparvazi 

Thanks for your suggestion!
The Developer Team will take it into consideration for future developments.
ping @NeOMakinG  : What do you think about this feature ?

Thanks!

**Comment by SharakPL** [comment]
Great suggestion! 👍

**Comment by alfonsolr09** [comment]
I think its a great suggestion too

**Comment by NeOMakinG** [comment]
If someone wants to work on it, the product team changed the status as Ready, we can implement it!

**Comment by marsaldev** [comment]
Hi @mparvazi,
would you still like to open a PR? 😊

**Comment by MWPROD11** [comment]
The project is already finished? I would love to implement it.

**Comment by jakub-przepiora** [comment]
i created other input to mobile, if you want that version i maked fork https://github.com/jakub-przepiora/ps_imageslider

**Comment by ButcherFromHell** [comment]
Hi,

I got confused here,  other input for mobile used to work but now (with 3.2.1) it is missing and couldn't find a word why is that. 

@jakub-przepiora it looks like your version is not working anymore with Presta 8.0.1. 

Thx.
